qualtran/surface_code/data_block.py

```diff
@@ -41,6 +43,10 @@ def footprint(self, n_algo_qubits: int) -> int:
     def data_error(self, n_algo_qubits: int, n_cycles: int, phys_err: float) -> float:
         """The error associated with storing data on `n_algo_qubits` for `n_cycles`."""
 
+    @abc.abstractmethod
+    def n_timesteps_to_consume_a_magic_state(self) -> float:
+        """The worst case number of timesteps needed to consume a magic state."""
```

Comment:
what is a timestep?

Reply:
![Screenshot from 2024-03-29 17-33-42](https://github.com/quantumlib/Qualtran/assets/5949112/1efc506e-c3cb-468d-bb70-861698cb4ca8)

Reply:
I changed timesteps to cycles and did the needed conversions for consistancy.